docs/easybuild-v5/overview-of-changes.md

```diff
@@ -48,7 +46,12 @@ feedback then please comment in the GitHub issue for the proposal.
 
 Various significant enhancements are included in EasyBuild v5.0, including:
 
-* *(no significant enhancements implemented so far in the `5.0.x` branches)*
+* [enable `--trace` by default][eb5_trace]
+
+### `--trace` enabled by default {: #eb5_trace }
+
+The [`--trace` option is enabled by default](https://github.com/easybuilders/easybuild-framework/pull/4250). This makes
+the steps and commands that EasyBuild is running clearer to the user.
```

Comment:
Should include a link to https://docs.easybuild.io/tracing-progress, and we should mention how to configure EasyBuild to not produce trace output, in case someone gets annoyed by it?

Reply:
added in 1b7cefd